contracts/erc721/InvoiceUnpaidNFT.sol

```diff
@@ -285,5 +285,31 @@ contract InvoiceUnpaidNFT is Initializable, UserMintableERC721 {
     );
   }
 
+  /**
+   * @dev Mints a token after validating the given merkle proofs
+   * and comparing it to the anchor registry's stored hash/doc ID.
+   * Only one more variable allowed
+   * @param to address The recipient of the minted token
+   * @param tokenId uint256 The ID for the minted token
+   * @param anchorId uint256 The ID of the document as identified
+   * by the set up anchorRegistry.
+   * using precise-proofs
+   */
+  function mint(
```

Comment:
Simplified public method and calling `internal` one. So we do not have to change the ABI (I hope)